File: extractor/models/data_row.py
class DataRow():
    def __init__(self, timestamp, viscosity, temperature):
        self.timestamp = timestamp
        self.viscosity = float(viscosity)
        self.temperature = float(temperature)

    def getTimestamp(self):
        return self.timestamp

    def getViscosity(self):
        return self.viscosity

    def getTemperature(self):
        return self.temperature

File: backend/modules/extractor/models/dataRow.js
class DataRow {
    constructor(timestamp, viscosity, temperature) {
        this.timestamp = timestamp;
        this.viscosity = viscosity;
        this.temperature = temperature;
    }

    getTimestamp() {
        return this.timestamp;
    }

    getViscosity() {
        return this.viscosity;
    }

    getTemperature() {
        return this.temperature;
    }
}

module.exports = {
    DataRow,
};